winforms/tests_backend/widgets/base.py

```diff
@@ -56,10 +57,30 @@ def color(self):
 
     @property
     def background_color(self):
-        if self.native.BackColor == Color.Transparent:
+        if (
+            self.widget.style.background_color is None
+            and self.native.BackColor.ToArgb() == SystemColors.Control.ToArgb()
+        ):
+            return None
+
+        if self.widget.style.background_color is TRANSPARENT and (
+            self.native.BackColor.ToArgb()
+            == self.widget.parent._impl.native.BackColor.ToArgb()
+        ):
             return TRANSPARENT
         else:
-            return toga_color(self.native.BackColor)
+            parent_color = toga_color(self.widget.parent._impl.native.BackColor)
+            blended_color = alpha_blending_over_operation(
+                self.widget.style.background_color, parent_color
+            )
```

Comment:
As above - the value of `background_color` shouldn't matter; we're trying to establish ground truth.